tuf/ngclient/updater.py

```diff
@@ -49,7 +49,8 @@
 
 from tuf.api import exceptions
 from tuf.api.metadata import Root, Snapshot, TargetFile, Targets, Timestamp
-from tuf.ngclient._internal import trusted_metadata_set, urllib3_fetcher
+from tuf.ngclient import urllib3_fetcher
```

Comment:
hmm, I guess it would be more appropriate to import straight from the sub module to minimize cycles in imports

Reply:
oops my mistake, that is the whole submodule being imported here, not just a class